cypress/e2e/article.spec.js

```diff
@@ -132,4 +132,64 @@ describe('Article JSON-LD', () => {
       });
     });
   });
+
+  it('matches schema when type LiveBlogPosting', () => {
+    Cypress.log({ message: 'azerty' });
+    cy.visit('http://localhost:3000/jsonld/liveBlogPosting');
+    cy.get('head script[type="application/ld+json"]').then(tags => {
+      const jsonLD = JSON.parse(tags[0].innerHTML);
+      assertSchema(schemas)('LiveBlogPosting', '1.0.0')(jsonLD);
+    });
+  });
+
+  it('renders with all props when type LiveBlogPosting', () => {
+    cy.visit('http://localhost:3000/jsonld/liveBlogPosting');
+    cy.get('head script[type="application/ld+json"]').then(tags => {
+      const jsonLD = JSON.parse(tags[0].innerHTML);
+      expect(jsonLD).to.deep.equal({
+        '@context': 'https://schema.org',
+        '@type': 'LiveBlogPosting',
+        datePublished: '2023-11-03T11:00:00+02:00',
+        description:
+          'Follow the announcements made during the 2023 Next.js Conf.',
+        mainEntityOfPage: {
+          '@type': 'WebPage',
+          '@id': 'https://example.com/liveblog',
+        },
+        headline: 'Next.js Conf live coverage',
+        image: [
+          'https://example.com/photos/1x1/photo.jpg',
+          'https://example.com/photos/4x3/photo.jpg',
+          'https://example.com/photos/16x9/photo.jpg',
+        ],
+        dateModified: '2023-11-03T19:00:00+02:00',
+        author: {
+          '@type': 'Person',
+          name: 'Jane Blogs',
+        },
+        coverageStartTime: '2023-11-03T11:00:00+02:00',
+        coverageEndTime: '2023-11-03T19:00:00+02:00',
+        liveBlogUpdate: [
```

Comment:
The `liveBlogUpdate` items could contain plenty of additional props because they are instances of [`BlogPosting`](https://schema.org/BlogPosting) but I decided to keep it short.